app/src/main/java/com/muzima/view/preferences/settings/SettingsPreferenceFragment.java

```diff
@@ -97,8 +97,20 @@ public boolean onPreferenceChange(final Preference preference, final Object newV
             @Override
             public boolean onPreferenceChange(Preference preference, Object o) {
                 Integer timeOutInMin = Integer.valueOf(o.toString());
-                ((MuzimaApplication) getActivity().getApplication()).resetTimer(timeOutInMin);
-                return true;
+                if (timeOutInMin > 0 && timeOutInMin < 500) {
```

Comment:
This looks very clean

Reply:
Adjust the PR name to match the ticket name of the ticket you are working on for Example: MUZIMA-111: Refactoring this and that...